### ISSUE TITLE
Using .hic/.cool files

### ISSUE THREAD

**Issue by aminakur** [vaquerizaslab/tadtool#26]
Is it possible to use .hic or .cool files as a matrix file?

**Comment by caragraduate** [comment]
Hi there, I am having the same question, and I wonder if you got this solved?

**Comment by aminakur** [comment]
No. I wonder how to convert .hic/.cool to the matrix + region file required by Tadtool.
